iNaturalist/src/main/res/values/do_not_translate.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="california_academy_of_sciences" translatable="false">California Academy of Sciences</string>
+    <string name="national_geographic" translatable="false">National Geographic</string>
+</resources>
```

Comment:
I know this is nitpicky, but could you have a newline at the end of the file here?

Reply:
Fixed. Apologies, I misunderstood.